Neos.Utility.Files/Classes/Files.php

```diff
@@ -246,7 +246,8 @@ public static function createDirectoryRecursively(string $path)
         if (is_file($path)) {
             throw new FilesException('Could not create directory "' . $path . '", because a file with that name exists!', 1349340620);
         }
-        if (!is_link($path) && !is_dir($path) && $path !== '') {
+        clearstatcache(true, $path);
```

Comment:
Shouldn't this be moved before the `is_file($path)` check above?

Reply:
Indeed, that would make sense.